client/app/scripts/liveblog-settings/styles/general-settings.less

```diff
@@ -0,0 +1,20 @@
+@PADDING: 20px;
+.general-settings {
```

Comment:
I agree, when we'll refactor to get a common service, let's find a way to fix this too